guides/micronaut-json-schema-generator/micronaut-json-schema-generator.adoc

```diff
@@ -0,0 +1,140 @@
+common:header-top.adoc[]
+
+== Getting Started
+
+This tutorial shows how to generate Java classes from https://json-schema.org/learn/getting-started-step-by-step[JSON schema] at compile-time using Micronaut JSON Schema.
```

Comment:
It is probably best to be consistent with naming. Use "JSON Schema" here.

Their official name is JSON Schema with second word capitalized.